src/features/content-insights/ContentInsightsSummary.tsx

```diff
@@ -8,18 +10,27 @@ import { GraphData } from './types';
 import './ContentInsightsSummary.scss';
 
 interface Props {
+    error: AxiosError | null;
     graphData: GraphData;
     isLoading: boolean;
     onClick: () => void;
     previousPeriodCount: number;
     totalCount: number;
 }
 
-const ContentInsightsSummary = ({ graphData, isLoading, previousPeriodCount, onClick, totalCount }: Props) => (
-    <div className="ContentInsightsSummary">
-        {isLoading ? (
-            <ContentInsightsSummaryGhostState />
-        ) : (
+const ContentInsightsSummary = ({ error, graphData, isLoading, previousPeriodCount, onClick, totalCount }: Props) => {
+    const errorStatus = !!error && !!error.response ? error.response.status : null;
```

Comment:
I don't think we need to overthink this at the moment -- just convert the error to a `isError` and show the error state. In a follow up we can probably pass the error into `ContentAnalyticsErrorState` and have that affect the message displayed